packages/react-button/src/components/MenuButton/MenuButton.types.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { ComposeOptions } from '@fluentui/react-compose';
+import { ComposeStandardStatics, ShorthandValue } from '../../utils/tempTypes';
+import { ButtonProps, ButtonTokens } from '../Button/Button.types';
+
+export interface MenuButtonProps extends Omit<ButtonProps, 'icon' | 'iconPosition' | 'loader'> {
+  /**
+   * Shorthand menu that is displayed when the button is pressed.
+   */
+  menu?: ShorthandValue<{}>;
```

Comment:
I added proper typing for `ShorthandValue` in `react-compose`. it's called `SlotProp`. pls use that and remove the ones from tempTypes